compute/oslogin/physical_security_key_ssh.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+#  Copyright 2022 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# [START compute_oslogin_physical_sk_script]
+import argparse
+import os
+import subprocess
+
+import googleapiclient.discovery
+
+
+def write_ssh_key_files(security_keys, directory):
+    """Store the SSH key files."""
+    key_files = []
+    for index, key in enumerate(security_keys):
+        key_file = os.path.join(directory, "google_sk_%s" % index)
+        with open(key_file, "w") as f:
+            f.write(key.get("privateKey"))
+            os.chmod(key_file, 0o600)
+            key_files.append(key_file)
+    return key_files
+
+
+def ssh_command(key_files, username, ip_address):
+    """Construct the SSH command for a given IP address and key files."""
+    command = ["ssh"]
+    for key_file in key_files:
+        command.extend(["-i", key_file])
+    command.append("{username}@{ip}".format(username=username, ip=ip_address))
+    return command
+
+
+def main(user_key, ip_address, dryrun, directory=None):
+    """Configure SSH key files and print SSH command."""
+    directory = directory or os.path.join(os.path.expanduser("~"), ".ssh")
+
+    # Create the OS Login API object.
+    oslogin = googleapiclient.discovery.build("oslogin", "v1beta")
+
+    # Retrieve security keys and OS Login username from a user's Google account.
+    profile = (
+        oslogin.users()
+        .getLoginProfile(name="users/{}".format(user_key), view="SECURITY_KEY")
+        .execute()
+    )
+    security_keys = profile.get("securityKeys")
+
+    if "posixAccounts" not in profile:
+        print("You need to have at least one OS Login enabled VM in your project.")
```

Comment:
If you don't have a posix account configured, that means the user hasn't configured a POSIX account. That doesn't have any relation to whether OS Login is enabled on a VM in the project.